analyzers/backend/docker/build.go

```diff
@@ -109,10 +110,13 @@ func checkResponse(rd io.Reader) error {
 		if err != nil {
 			return err
 		}
-		if d.Stream == "" || d.Stream == "\n" || strings.TrimSuffix(d.Stream, "\n") == currentStream {
+		if d.Stream == "" || d.Stream == "\n" || strings.Contains(d.Stream, "--->") || strings.TrimSuffix(d.Stream, "\n") == currentStream {
 			continue
 		}
 		currentStream = strings.TrimSuffix(d.Stream, "\n")
+		if showAllLogs {
```

Comment:
Wouldn't it be better to return the reader, and get the client of this API (the `command/analyzer/verify` package) to print the output however it wants?